environments/notifications/in-app-messaging/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "in-app-messaging",
```

Comment:
```suggestion
  "name": "in-app-messaging"-environment,
```
for consistency with other environments.

Reply:
Fixed in latest commit